src/mpid/ch4/ch4_api.txt

```diff
@@ -86,7 +86,7 @@ Non Native API:
       NM*: rank, comm, handler_id, tag, buf-2, count, datatype, src_vci, dst_vci, rreq
      SHM*: rank, comm, handler_id, tag, buf-2, count, datatype, src_vci, dst_vci, rreq
   comm_set_vcis : int
-      NM : comm, num_vcis, all_num_vcis
+      NM : comm, num_implicit, num_reserved, all_num_vcis
      SHM : comm, num_vcis
```

Comment:
will we need any changes to shm vcis along these lines?

Reply:
The reason for `NM` to have those parameters and output `all_num_vcis` is because netmod decides how many actual vcis it can support. For `SHM`, it is an input only parameter.

Reply:
Actually, `SHM` does even need `num_vcis`. It takes settings from `MPIDI_global`.